contracts/Bridge.sol

```diff
@@ -186,7 +186,7 @@ contract Bridge is Pausable, AccessControl, SafeMath {
         @param relayerAddress Address of relayer to be removed.
         @notice Emits {RelayerRemoved} event.
      */
-    function adminRemoveRelayer(address relayerAddress) external onlyAdmin {
+    function adminRemoveRelayer(address relayerAddress) external onlyAdmin { // REVIEW: onlyAdmin not needed as it is checked in the revokeRole.
```

Comment:
:+1: awesome, we'll be able to reduce the inlining of some code with the two suggestions